src/activity.js

```diff
@@ -15,7 +15,6 @@ class Activity {
 
     markComplete() {
         this.completed = true;
```

Comment:
I'm still wondering about this function. I'm not sure we've done enough with our data model in this area.

Reply:
hmm, i am actually not sure what you are referring to here with this.completed. I see you create an instance of an activity class, are you looking to set completed property on that object to true?

Reply:
to me its seems like you might want to refer to that instance.

Reply:
This function seems intuitive enough IMO - updates the instance property `completed` to `true`. @ttarlov  - this function is a class method, available to the instance.

Reply:
my bad guys. I see that you invoke this method on like 190 in main. Its all good. had to pull down the branch to make this more clear.